### ISSUE TITLE
9.3.3.1 Fehlererkennung - mögliche Änderung der Zuordnung programmatischer Mängel

### ISSUE THREAD

**Issue by detlevhfischer** [BIK-BITV/BIK-Web-Test#421]
Hallo,
bisher erfassen wir im BIK BITV-Test programmatische Mängel, etwa die fehlende Verknüpfung von inline-Fehlermeldungen über Einschluss ins Label oder Verweis mittels `aria-describedby`, unter dem Prüfschritt 9.3.3.1 Fehlererkennung.

Diskutiert wird im 2.X "Aufräumteam" der Working Group gerade ein Pull Request #1803 [Update/correct 3.3.1 understanding](https://github.com/w3c/wcag/pull/1803#top)

...der nahe legt, dass diese Mängel bei 9.4.1.2 Name, Rolle, Wert erfasst werden sollten - ebenso die fehlende programmatische Identifikation von Pflichtfeldern, wenn diese visuell verfügbar ist (9.1.3.1 oder auch 9.4.1.2, das war mi nicht ganz klar).

Ich habe [in meinem Review des PR](https://github.com/w3c/wcag/pull/1803#pullrequestreview-1887921857) einigen Bedenken Ausdruck verliehen. Je nach Working Group Entscheidung kann das bei uns einige Änderungen der Zuordnung von Mängeln zu Prüfschritten nach sich ziehen.

**Comment by detlevhfischer** [comment]
Der [überarbeitete Understanding-Text](https://www.w3.org/WAI/WCAG22/Understanding/error-identification.html) erwähnt, das die programmatische Ermittelbarkeit ergänzend hinzukommen kann, aber im Rahmen von 3.3.1 nicht verpflichtend ist:
> The identification and description of an error can be combined with programmatic information that user agents or assistive technologies can use to identify an error and provide error information to the user. For example, certain technologies can specify that the user's input must not fall outside a specific range, or that a form field is required. Currently, few technologies support this kind of programmatic information, but this success criterion does not require, nor prevent it.

Aus thematischer Sicht finde ich es weiter sinnvoll, den Hinweis auf eine fehlende programmatische Ermittelbarkeit von Fehlermeldungen in diesem Prüfschritt zu lassen, aber im Sinne der Bewertung passt es tatsächlich besser zu 4.1.2 bzw. 1.3.1h. Klar scheint, dass allein die fehlende programmatrische Verknüpfung schriftlicher spezifischer Fehlermeldungen kein FAIL von 3.3.1 rechtfertigen würde.
